File: src/main/java/org/pixeltime/enchantmentsenhance/commands/VersionCommand.java
package org.pixeltime.enchantmentsenhance.commands;

import org.bukkit.entity.Player;
import org.pixeltime.enchantmentsenhance.manager.SettingsManager;

public class VersionCommand extends SubCommand {

    @Override
    public void onCommand(Player player, String[] args) {
        new VersionCommand().onCommand(player, args);
    }


    @Override
    public String name() {
        return "version";
    }


    @Override
    public String info() {
        return "\n&6/enhance version &7- " + SettingsManager.lang.getString(
                "Help.version");
    }


    @Override
    public String[] aliases() {
        return new String[]{"version", "ver", "banben", "bb"};
    }


    @Override
    public String getPermission() {
        return "Enchantmentsenhance.version";
    }

}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/command/player/VersionCommand.kt
package org.pixeltime.enchantmentsenhance.command.player

import org.bukkit.entity.Player
import org.pixeltime.enchantmentsenhance.command.SubCommand
import org.pixeltime.enchantmentsenhance.command.console.VersionConsoleCommand

class VersionCommand : SubCommand() {


    override val permission: String
        get() = "Enchantmentsenhance.version"

    override fun onCommand(player: Player, args: Array<String>) {
        VersionConsoleCommand().onCommand(player, args)
    }


    override fun name(): String {
        return "version"
    }


    override fun usage(): String {
        return "/enhance version"
    }


    override fun aliases(): Array<String> {
        return arrayOf("version", "ver", "banben", "bb")
    }

}
